tests/improver-nbhood/00-null.bats

```diff
@@ -36,7 +36,7 @@
 usage: improver-nbhood [-h]
                        [--radius-in-km RADIUS | --radii-in-km-by-lead-time \
 RADIUS_BY_LEAD_TIME RADIUS_BY_LEAD_TIME]
```

Comment:
Good point. I hadn't realise I could specify the `metavar` option like that, so thanks!